src/components/SideBar/index.tsx

```diff
@@ -1,14 +1,59 @@
 import React from 'react';
 
 import './styles.scss';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+import { AppState } from '../../redux/reducer';
+import { AppAction, SectionName } from '../../redux/actionTypes';
+import { changeSection } from '../../redux/actions';
+import { DispatchProps } from '../dispatchProps';
```

Comment:
we should sort these imports alphabetically and import libraries first and then our own classes